templates/recording-deployment.yaml

```diff
@@ -81,15 +82,63 @@ spec:
           name: tmp
         resources:
           limits:
-            cpu: {{ .Values.recording.cpu | default "1000m"}}
-            memory: {{ .Values.recording.memory | default "1G"}}
+            cpu: {{ .Values.recording.cpu | default "256m"}}
+            memory: {{ .Values.recording.memory | default "256Mi"}}
           requests:
-            cpu: {{ .Values.recording.cpu | default "1000m"}}
-            memory: {{ .Values.recording.memory | default "1G"}}
+            cpu: {{ .Values.recording.cpu | default "256m"}}
+            memory: {{ .Values.recording.memory | default "256Mi"}}
         securityContext:
           {{ include "defaultSecurityContext" . | nindent 10 }}
           runAsGroup: {{ (.Values.recording.runAs).group | default "1000" }}
           runAsUser: {{ (.Values.recording.runAs).user | default "1000" }}
+      # Chromium container
+      - envFrom:
+        - configMapRef:
+            name: {{ .Release.Name }}-chromium-envvars
+        - configMapRef:
+            name: {{ .Release.Name }}-chromium-custom-envvars
+            optional: true
+        image: {{ (.Values.chromium.image).ref | default "ghcr.io/cobrowseio/cobrowse-chromium-enterprise:1.3.12" }}
```

Comment:
A little bit of hand-waving here in terms of how we release/version this. Right now it's kind of an "extra" in the recording image build. Will still need to do more in terms of release scripts. I'm kind of assuming we'll release a new chromium container each time we release a new recording container, and the versions will match.